tests/checkers/inference/test/CFInferenceTest.java

```diff
@@ -39,6 +39,10 @@ public String getPathToAfuScripts() {
         return System.getProperty("path.afu.scripts");
     }
 
+    public String getPathToInferenceScript() {
+        return "./scripts/inference";
```

Comment:
To get the AFU script path, we are using `System.getProperty("path.afu.scripts")` instead of overriding the getter method  in downstream type system. How about we set a system variable to include the inference script and read the variable when we need to script location? If we use method like this, then every downstream type system, eg. Dataflow, Ontology, GUT, needs to override this getter method to point to the inference script, which is like [this](https://github.com/opprop/generic-type-inference-solver/blob/a6b946f815ca17e2461ec8ebea94ae77610505a1/testing/tests/DataflowTest.java#L26).